content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -6,68 +6,87 @@ metaDescription: 'Learn how to deploy Node.js and TypeScript applications that a
 
-Go to [`deployment-example-netlify`](https://github.com/prisma/deployment-example-netlify) and fork the repository by clicking the **Fork** button in the top-right corner:
+To begin, fork the repository of the deployment example project [`deployment-example-netlify`](https://github.com/prisma/deployment-example-netlify).
+
+With a forked repository, you create a copy of the repository to which you will be able to commit your own changes. Later, you will connect the forked repository to Netlify so that changes are automatically built and deployed.
+
+### Steps
+
+1. Go to [`deployment-example-netlify`](https://github.com/prisma/deployment-example-netlify) and click the **Fork** button in the top-right corner to fork the repository:
+
+   ![GitHub repository - click Fork button](images/500-02-deploying-to-netlify-example-repo-click-fork.png)
 
-![fork button](https://res.cloudinary.com/prismaio/image/upload/v1628765101/docs/nyap5pW_g0tbuw.png)
+2. On the **Create a new fork** page, configure the forked repository.
+   1. (Optional) From **Owner**, select a GitHub account or organization as the owner.
+   2. (Optional) In **Repository name**, modify the repository name if necessary.
+   3. Leave as selected **Copy the `main` branch only**.
+3. Click **Create fork**.
+   ![GitHub - Create a new fork page](images/500-03-deploying-to-netlify-example-repo-create-fork-page.png)
 
-By forking, you create a copy of the repository to which you will be able to commit your changes. Later in the guide, you will connect the forked repository to Netlify so that changes are automatically built and deployed.
+### Result
 
-**Checkpoint:** After the forking is complete, you should see the forked repository in your account, for example `https://github.com/YOUR_GITHUB_USERNAME/deployment-example-netlify`
+After the forking is complete, you should see the forked repository in your account, for example `https://github.com/YOUR_GITHUB_USERNAME/deployment-example-netlify`.
 
 ## 2. Clone the fork and install dependencies
 
-Open your terminal, navigate to a location of your choice and clone your fork of the `deployment-example-netlify`:
+You need to clone the repository to a folder on your local filesystem to install all required npm packages.
 
-```terminal
-git clone git@github.com:YOUR_GITHUB_USERNAME/deployment-example-netlify.git
-```
+### Steps
 
-After cloning the repository, install the dependencies:
+1. Open your terminal, navigate to a location of your choice and clone your fork of the `deployment-example-netlify`:
 
-```terminal
-cd deployment-example-netlify
-npm install
-```
+   ```terminal
+   git clone git@github.com:YOUR_GITHUB_USERNAME/deployment-example-netlify.git
+   ```
 
-**Checkpoint:** `ls -1` should show:
+2. After cloning the repository, install the dependencies:
+
+   ```terminal
+   cd deployment-example-netlify
+   npm install
+   ```
+
+### Result
+
+When you run `ls -1` from the cloned folder, you should see the following result.
```

Comment:
Note sure if the specific command is relevant here. I would assume developers knowing how to figure out which files and folders are present. I think just describing that this is what the content of the folder should look like now should be fine.

(`ls` does not work on Windows of course)